gen_docker_compose.py

```diff
@@ -45,6 +47,16 @@ def __init__(self, templates_folder: Union[str, None] = None):
         config.read('settings.ini')
         self.config = config
 
+    def short_hash(self) -> str:
+        short_hash = '1234567'
```

Comment:
Personally, I'd prefer something like `0000000` or `fffffff`. It is easier to catch with a glance.

Reply:
was difficult to count, I don't have any issues if I push another version. I will update.